docs/_includes/calendar.html

```diff
@@ -0,0 +1,37 @@
+<div id="calendar">
+    {% include section-header.html name="calendar" version=page.versions.calendar %}
+
+    <p>The calendar component is used as a building block for other components, such as date picker.</p>
+
+    <h3>Standard Calendar</h3>
+
+    <p>This component will stretch to the width of its container.</p>
+
+    {% include calendar-month.html %}
+    {% highlight html %}
+{% include calendar-month.html %}
+    {% endhighlight %}
+
+    <h3>Stacked Calendars</h3>
+
+    <p>This is a demonstration of a context in which the calendar module may be used.</p>
+
+    <div style="width: 311px">
+        {% include calendar-month.html index="1" %}
```

Comment:
I see the stacker calendar show a cell containing a button. Is this intentional at this stage of the docs?